styles.css

```diff
@@ -454,415 +527,470 @@ body {
     display: none;
   }
 
-  .drop-down-menu {
-    z-index: 10;
-    color: #fff;
-    background-color: #6070ff;
-    height: 110vh;
-    width: 100%;
-    mix-blend-mode: multiply;
-    backdrop-filter: blur(7px);
-    position: fixed;
-  }
-
-  .hidden-menu {
+  .desktop-work {
     display: none;
   }
 
-  .hamburger-icon {
-    margin-right: 5%;
-    margin-top: 3%;
+  .mobile-popup {
+    height: 90%;
+    width: 80%;
+    background: #fff;
+    z-index: 2;
+    position: fixed;
+    top: 0;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5%;
+    margin: 5%;
+    margin-top: 10%;
   }
 
-  .show {
-    display: block;
+  .popup-line {
+    width: 90%;
+    height: 2px;
+    background-color: #ebecf0;
+    margin: 5%;
   }
```

Comment:
- Great job for implementing mobile popups 👍👍, I think it would be better if you make the background blur and update the `see live` and `see source` buttons to stick the Figma design and have better look👍👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/218041851-d5cdb884-6b2c-4cca-bd92-e33fe2647163.png" width="300">|<img src="https://user-images.githubusercontent.com/52010789/218041855-74427801-679c-48b5-b5e9-068d3a895904.png" width="300">|


😎️

🤝️

Reply:
- [ ] Great work making some adjustments to the button. To make the buttons look identical, kindly add some padding to the _**See live**_ and _**See Source**_ buttons.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/218090961-361566cc-4570-4df1-99f8-5adfa303a9c3.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/218091039-e7cce4da-441a-4ec6-a28f-516853a46879.png)